### ISSUE TITLE
step: removing 5 features in each iteration

### ISSUE THREAD

**Issue by mubashirtuf** [DistrictDataLabs/yellowbrick#664]
Using RFECV, the graph shows best features with one increment in each iteration.
I want to show each iteration should work with an increment of 5 features.
Example: score  should be plotted against x-axis of [5,10,15,20,25].
I used step=5, but its not working.

**Comment by bbengfort** [comment]
@mubashirtuf I'm sorry it's taken so long for us to respond; things got busy at the end of the semester and during the holiday. I can take a look at this, but it would be helpful if you could provide the code you used to generate the error - on my quick first attempt I couldn't reproduce the issue. I see a place where the problem might be occurring, but I'm not totally sure. Would you also upload the figure that was produced?

**Comment by bbengfort** [comment]
@mubashirtuf ok, I figured out the issue! 

Here is the example code I used to reproduce:

```python
from sklearn.datasets import make_classification
from sklearn.linear_model import LogisticRegression
from yellowbrick.features import RFECV 

X, y = make_classification(
    n_samples=200, n_features=30, n_informative=18, n_redundant=6,
    n_repeated=0, n_classes=8, n_clusters_per_class=1, random_state=0
)

oz = RFECV(LogisticRegression(), step=5).fit(X, y)
oz.poof()
```

This resulted in the following image:

![example_step_bug_rfecv](https://user-images.githubusercontent.com/745966/50542371-3be88f80-0b8a-11e9-896d-4e3f9b1d630a.png)

Although step is passed to the `RFE` estimator to eliminate that many features each time, the CV methods are still using a step of 1. I fixed this and now the output looks as follows:

![example_stepped_rfecv](https://user-images.githubusercontent.com/745966/50542380-72bea580-0b8a-11e9-8425-926cdf534f12.png)

I've created a fix for this in #679 and it should be available in version 1.0 -- thank you for sending us the bug report!

**Comment by bbengfort** [comment]
@mubashirtuf this is fixed in the develop version of Yellowbrick and will be available in Yellowbrick v1.0 -- in the meantime you can `pip install git+https://github.com/DistrictDataLabs/yellowbrick` to get the latest feature. Thank you again for contributing to yellowbrick!
